compiler/rustc_trait_selection/src/traits/wf.rs

```diff
@@ -372,14 +378,18 @@ impl<'a, 'tcx> WfPredicates<'a, 'tcx> {
     }
 
     /// Pushes the obligations required for `trait_ref` to be WF into `self.out`.
-    fn compute_trait_pred(&mut self, trait_pred: ty::TraitPredicate<'tcx>, elaborate: Elaborate) {
+    fn add_trait_pred_wf_reqs(
```

Comment:
Nitpick: I think I'd prefer a naming scheme like `add_wf_reqs_for_X` over `add_X_wf_reqs`. A "suffix style" naming scheme means that they're all grouped together when you grep/ctrl+F for `add_wf_reqs_for`, for example.

Reply:
Another micro nitpick that I care a bit less about than above is "`reqs`". I don't think we use that particular shortening anywhere else in the compiler -- I'd like rather we call them like... `wf_preds` or `wf_goals` or just `wf_requirements` or something.

Reply:
:thinking: yeah thats a good idea